.env-devel

```diff
@@ -58,6 +58,8 @@ DYNAMIC_SIDECAR_PROMETHEUS_MONITORING_NETWORKS=[]
 
 FUNCTION_SERVICES_AUTHORS='{"UN": {"name": "Unknown", "email": "unknown@osparc.io", "affiliation": "unknown"}}'
 
+MONITORING_ENABLED=True
```

Comment:
AFAIK you should also add this in all configs

This is a part of the instructions by @YuryHrytsuk 
![image](https://github.com/ITISFoundation/osparc-simcore/assets/32402063/2a5dcf7a-9477-43da-b028-c97da033f233)

I could not see at least `MONITORING_ENABLED=True` there

Reply:
Yes, removed it already